packages/api/src/models/medical/organization.ts

```diff
@@ -35,6 +37,14 @@ export class OrganizationModel extends BaseModel<OrganizationModel> implements O
         data: {
           type: DataTypes.JSONB,
         },
+        cqActive: {
+          type: DataTypes.BOOLEAN,
+          defaultValue: false,
+        },
+        cwActive: {
+          type: DataTypes.BOOLEAN,
+          defaultValue: false,
+        },
```

Comment:
what do these mean, and why are they necessary when they weren't previously?

Reply:
These are the active / not-active status of a Commonwell / Carequality organizations (same ones as on the facility table). They're not "necessary" but they help us keep the respect HIE entities accurate without having to fetch the existing entity from the HIE on every update.

Right now, there's no "update a single field" for either CQ or CW, which means we rebuild the entity each time, even for updates. This means if only one field changes, we still need to know the current values of the rest of the fields. Pretty much all the "dynamic" fields for CQ / CW are stored in RDS already in organization / facility (i.e. oid, addresses, names, org biz type, org type, etc.) but not whether or not the CW / CW org is active.

We *do* have these fields on the facility table, but not organization table (hence added here).